src/cmd/cmdfuel.rb.in

```diff
@@ -156,7 +157,11 @@ SUBCOMMANDS = {
   "  -p [--private]           Use this argument to make the model private. \n"\
   "                           Otherwise, the model will be public.         \n"\
   "  --header arg             Set an HTTP header, such as                  \n"\
-  "                           --header 'authorization: Bearer JWT'.        \n" +
+  "                           --header 'Private-Token: <access_token>'.    \n"\
```

Comment:
This is the part that needs to move  up to the `download` section. I think you can wait on PR #116.